app/src/main/java/com/codepath/earthquakemonitor/LoginActivity.java

```diff
@@ -56,4 +77,26 @@ public void onClick(View view)
 
     }
 
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
```

Comment:
Do we really need to call the superclass method here?

Reply:
I tried remove it but still the break point in done() method didnt stop, it only stops when I cancel login, but if login success it does nothing. So I just keep it there ...

Reply:
ok! you have merge conflicts now :(